src/styles/index.scss

```diff
@@ -1,3 +1,80 @@
+$font-family: Arial, sans-serif;
+$font-size: 30px;
+$day-border: 1px;
+$day-size: 100px;
+$day-color: #eee;
+$hover-color: #FFBFCB;
+$animation-duration: 0.5ms;
```

Comment:
The unit for the animation duration should be 's' for seconds or 'ms' for milliseconds. '0.5ms' is likely a typo and should be '0.5s' if you meant half a second.